Automattic/wp-calypso
# Fixes an error with redux in the Blaze Dashboard app

## Proposed Changes

This PR fixes a redux-related error with the Blaze Dashboard app that is thrown when it starts.

![Screenshot 2023-06-01 at 14 35 21](https://github.com/Automattic/wp-calypso/assets/1258162/e6547a17-2726-4dfd-9232-21519bf5d66f)

The problem was because of outdated libraries on the package.json

## Testing Instructions
1. Check out this branch locally
2. Check out this Jetpack branch: https://github.com/Automattic/jetpack/pull/30103
3. In the console, go to `apps/blaze-dashboard` and execute: 
```
yarn install
BLAZE_DASHBOARD_PACKAGE_PATH=/path/to/jetpack/projects/packages/blaze yarn dev
```
**Note:** Change the jetpack path to the correct folder installation.

4. Build the Jetpack Blaze package: `pnpm jetpack build packages/blaze`
5. Open your Jurrasic Tube tunnel and then in the browser go to the wp-admin page for your local site
**Note:** You need to enable this filter `jetpack_blaze_dashboard_enable` inside your local Wordpress installation.
You can ping me if you don't know how to do that.

6. Click on Tools->Advertising
7. Verify that the Dashboard is displayed correctly.

## Pre-merge Checklist
